### ISSUE TITLE
Build fails with missing vkd3d_windows.h

### ISSUE THREAD

**Issue by adolfintel** [HansKristian-Work/vkd3d-proton#1487]
I'm sure you've already noticed, but vkd3d-proton builds have been failing for a couple days.

I get the same error on my PC as the CI on github:
```
[24/125] Generating 'libs/vkd3d-common/libvkd3d_common.a.p/vkd3d_d3d12.h'.
FAILED: libs/vkd3d-common/libvkd3d_common.a.p/vkd3d_d3d12.h 
/usr/bin/widl -h -o libs/vkd3d-common/libvkd3d_common.a.p/vkd3d_d3d12.h ../../../vkd3d-proton/include/vkd3d_d3d12.idl
stdin:19: error: Unable to open include file vkd3d_windows.h
[41/125] Compiling C++ object subprojects/d...rv/libdxil-spirv.a.p/dxil_converter.cpp.obj
ninja: build stopped: subcommand failed.
```

This has been going on for a couple days now, since d3d12core was introduced I think.

**Comment by HansKristian-Work** [comment]
It's a Wine 8.4 issue and it's known. Not our bug.

**Comment by Bananaman** [comment]
@HansKristian-Work Seems like this is it, and if so it will be fixed in Wine 8.5:

https://gitlab.winehq.org/wine/wine/-/commit/78dc4ce7a0eed9c0bdeeea95e4ce895582eca4df
